CITATION.cff

```diff
@@ -27,6 +27,7 @@ contact:
 license: BSD-4-Clause
 
 authors:
+  - family-names: Nilearn contributors
```

Comment:
@emdupre I went with the first thing that crossed my mind but happy to use anything else.

Reply:
I think that's fine ! I'd say "developers", but I don't think this word choice is going to make or break the consortium idea :)

Reply:
I don't think so either. And we still have some time before the release in case we got a change of heart